security/fma-shared-lockbox.md

```diff
@@ -0,0 +1,147 @@
+- **Description:** If an attacker bypasses access controls for `unlockETH`, the deposit and withdrawal invariants can be broken, potentially resulting in the worst-case scenario of draining all ETH liquidity from the `SharedLockbox`.
+- **Risk Assessment:** High.
+    - Potential Impact: Critical. Unauthorized ETH unlocking would result in a direct loss of ETH from the `SharedLockbox`.
+    - Likelihood: Low. Access controls are specified in `if (!superchainConfig().authorizedPortals(msg.sender)) revert Unauthorized()`, but errors or misconfigurations remain possible.
+- **Mitigation:** There should be tests for `lockETH` and `unlockETH` checks. Access control must be strictly enforced, permitting only approved `OptimismPortal` contracts via the `authorizedPortals` mapping in `SuperchainConfigInterop`.
+- **Detection:** Monitor `ETHUnlocked` events to verify consistency with authorized portal addresses. Set up alerts for suspected unauthorized activity from non-approved addresses.
+- **Recovery Path(s):** Pause the system through `SuperchainConfig` to halt the `SharedLockbox` upon detection of unauthorized access.
+
+### FM2: Compromised or poorly implemented `clusterManager` on L1
+
+- **Description:** The `clusterManager` role in `SuperchainConfigInterop` can directly call `addDependency` on L1, bypassing the normal L2 → L1 flow. If compromised, an attacker could add malicious or arbitrary chain IDs and authorize their own portal.
+- **Risk Assessment:** High.
+    - Potential Impact: Critical. A malicious addition to the dependency set could allow an attacker to `unlockETH` from the `SharedLockbox`.
+    - Likelihood: Low. The `clusterManager` should be a highly secured role (e.g., a multi-sig or governance key).
+- **Mitigations:** Tests to ensure `addDependency` cannot be called by unauthorized accounts. Strictly secure the `clusterManager` key.
+- **Detection:** Track the `DependencyAdded` event for calls from `clusterManager`. Compare newly added chain IDs to an approved list.
+- **Recovery Path(s):** Pause the system through `SuperchainConfig`. Since removal functions are unavailable, an upgrade directly over **`SuperchainConfigInterop`** to clear storage slots will be required.
+
+### FM3: Unauthorized or incorrectly initialized `superchainConfig` address in `OptimismPortalInterop`.
+
+- **Description:** In both `SharedLockbox` and `OptimismPortalInterop`, the address of the `SuperchainConfig` is stored in a special storage slot. If it’s set incorrectly or to a malicious address, the entire access-control scheme breaks.
+- **Risk Assessment:** High.
+    - Potential Impact: Critical. Since the entire access-control scheme depends on `SuperchainConfig`, all funds could potentially be extracted from the **OptimismPortal**.
+    - Likelihood: Low. Storage slots and values are carefully set during protocol upgrades.
+- **Mitigations:** There should be tests for the initialization step, and ensuring that `superchainConfig` is only set once.
```

Comment:
I would add here that the security team must validate "checklist style" that this storage slot is not changed unintentionally before any upgrade.